.gitlab-ci.yml

```diff
@@ -237,6 +243,37 @@ gitleaks:
   script: gitleaks detect -v --source .
 
 
+## Jobs running on Slurm cluster
+
+# factored out command to download the datasets, cmake, and build in non-debug mode
+.setup_and_build_ndebug: &setup_and_build_ndebug
+  - ./tools/downloadDatasets.sh <<< "yes"
```

Comment:
does it mean it re-downloads datasets for every submit? If yes, the datasets are also on `/storage/datasets` that are available on the cluster, and may be better to just create a symlink to there instead of calling this script?